plugins/com.google.cloud.tools.eclipse.appengine.deploy/src/com/google/cloud/tools/eclipse/appengine/deploy/AppEngineProjectDeployer.java

```diff
@@ -35,21 +40,43 @@
  */
 public class AppEngineProjectDeployer {
 
+  @VisibleForTesting
+  static final List<String> APP_ENGINE_CONFIG_FILES = Collections.unmodifiableList(Arrays.asList(
```

Comment:
Why does this not include app.yaml? Why do we always deploy that and not deploy these? Until we enable one-by-one deploy, I think we ought to just deploy everything, no checkbox.